tests/cppException/enc/CMakeLists.txt

```diff
@@ -1,14 +1,16 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-include(add_compile_flags_if_supported)
-
-include(add_enclave_executable)
-add_executable(cppException_enc enc.cpp cppException.cpp)
+add_executable(cppException_enc
+  enc.cpp
+  cppException.cpp)
 
 target_compile_features(cppException_enc PRIVATE cxx_generalized_initializers)
-add_target_compile_flags_if_supported(cppException_enc PRIVATE CXX
-    -Wno-error=exceptions
-    -Wno-error=terminate
```

Comment:
I could be wrong, but this did not appear to be valid for neither GCC 5.4 nor Clang 7.

Reply:
Just a word of caution -- this is an exception test and so please step through the code and verify that you are not getting a False PASS or breaking the test intent.

Reply:
I didn't change the logic here. This is just disabling a warning for a certain compiler.

Reply:
ok.

Reply:
I'm fairly confident that the only thing this could break is the build (by failing due to `-Werror`) and that's taken care of; is there anything in particular to test other than building and running the full test suite?